src/main/java/com/books/BookApplication.java

```diff
@@ -3,12 +3,17 @@
 import com.books.entities.Book;
 import com.books.entities.Person;
 import com.books.entities.Publisher;
+import com.books.services.AuthorService;
 import com.books.services.BookService;
+import com.books.services.EncryptService;
+import com.books.storage.abstracts.Repository;
+import com.books.storage.concrete.SQL.BookSQLRepository;
+import com.books.utils.DatabaseCreator;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.FileNotFoundException;
 import java.util.ArrayList;
-import java.util.Collection;
 import java.util.Date;
 import java.util.List;
 
```

Comment:
I think that for now this main class is redundant. Your application is starting up without using main class. Also you should make respective changes in another parts of code( E.g. remove jar-plugin because you have no executive jar with main class at this moment).